### PR TITLE
In WebIDL, make all layer initializations optional

### PR DESCRIPTION
**Pull request by asajeffrey**
Make all the `init` parameters for layer creation optional.

### CODE REVIEW

**Comment by cabanier** [comment]
Thanks @asajeffrey !

Can you make sure your GitHub account is linked to the Mozilla org so that the ipr process is happy with the change? [Details here](https://labs.w3.org/repo-manager/pr/id/immersive-web/layers/56)

**Comment by asajeffrey** [comment]
I linked my Github to my W3C account, and my W3C account is linked to Mozilla, does the check need to be rerun?

**Comment by cabanier** [comment]
> I linked my Github to my W3C account, and my W3C account is linked to Mozilla, does the check need to be rerun?

There was a revalidate button and that fixed it

File: README.md [hunk 1]
```diff
@@ -367,11 +367,11 @@ interface XRWebGLBinding {
 
   readonly attribute double nativeProjectionScaleFactor;
 
-  XRProjectionLayer createProjectionLayer(XRTextureType textureType, XRProjectionLayerInit init);
-  XRQuadLayer createQuadLayer(XRTextureType textureType, XRLayerInit init);
-  XRCylinderLayer createCylinderLayer(XRTextureType textureType, XRLayerInit init);
-  XREquirectLayer createEquirectLayer(XRTextureType textureType, XRLayerInit init);
-  XRCubeLayer createCubeLayer(XRLayerInit init);
+  XRProjectionLayer createProjectionLayer(XRTextureType textureType, optional XRProjectionLayerInit init = {});
+  XRQuadLayer createQuadLayer(XRTextureType textureType, optional XRLayerInit init = {});
```
Comment (cabanier) [inline README.md:371]: Only XRProjectionLayerInit has all default parameters so the init can be omitted. XRLayerInit needs a pixel width and height.

Comment (asajeffrey) [reply README.md]: Oops, good point. Fixed.

**Pull request merged**